doc/rados/operations/health-checks.rst

```diff
@@ -285,28 +285,29 @@ OSDs
 OSD_DOWN
 ________
 
-One or more OSDs are marked down.  The ceph-osd daemon may have been stopped,
-or peer OSDs may be unable to reach the OSD over the network.  Common causes
-include a stopped or crashed daemon, a down host, or a network outage.
+One or more OSDs are marked "down". The ceph-osd daemon might have been
```

Comment:
backticks

Reply:
I think this is a mention, not a use. This, however, is part of a use-mention confusion that is general throughout the documentation, and I will bear it in mind.